src/websocketService/codes.ts

```diff
@@ -0,0 +1,91 @@
+// Defines constants for all websocket close event codes that are used in practice
+// Link: https://developer.mozilla.org/en-US/docs/Web/API/CloseEvent
+
+/**
+ * Normal closure; the connection successfully completed whatever purpose for which it was created.
+ */
+export const NORMAL_CLOSURE = 1000;
```

Comment:
I can imagine that there is a module for that that predefines these constant, but haven't checked

Reply:
I can't easily find any, this is what I found of someones gist
https://gist.github.com/RobertMenke/443e5cc57e4569c48b08c9592740da77

Reply:
👍🏼